l10n_ir_base/models/account_journal_dashboard.py

```diff
@@ -0,0 +1,71 @@
+from datetime import date, datetime, timedelta
+from odoo.api import model
+import jdatetime as jd
+
+
+from babel.dates import format_datetime, format_date
+from odoo import models, api, _, fields
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT as DF
+from odoo.tools.misc import formatLang, format_date as odoo_format_date, get_lang
+import random
+
+class account_journal(models.Model):
+    _inherit = "account.journal"
+
+    def get_bar_graph_datas(self):
+        data = []
```

Comment:
Concern: I don't see a condition to apply this logic only if Jalali is enabled.
Users are expected to be able to choose the language and date format they want to use.
I don't see that here.

Reply:
This section was added